tests/accumulator_test.py

```diff
@@ -262,18 +263,60 @@ def test_without_noise(self):
 class SumAccumulatorTest(unittest.TestCase):
 
     def test_without_noise(self):
+        no_noise = MeanVarParams(eps=1,
+                                 delta=1,
+                                 low=0,
+                                 high=0,
+                                 max_partitions_contributed=1,
+                                 max_contributions_per_partition=1,
+                                 noise_kind=pipeline_dp.NoiseKind.GAUSSIAN)
         sum_accumulator = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(6)))
+            accumulator.SumParams(no_noise), list(range(6)))
         self.assertEqual(sum_accumulator.compute_metrics(), 15)
 
         sum_accumulator.add_value(5)
         self.assertEqual(sum_accumulator.compute_metrics(), 20)
 
         sum_accumulator_2 = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(3)))
+            accumulator.SumParams(no_noise), list(range(3)))
         sum_accumulator.add_accumulator(sum_accumulator_2)
         self.assertEqual(sum_accumulator.compute_metrics(), 23)
 
+    def test_with_noise(self):
+        sum_accumulator = accumulator.SumAccumulator(
+            accumulator.SumParams(
+                MeanVarParams(eps=100,
+                              delta=1,
```

Comment:
Let's use the following parameters
eps = 10
delta = 1e-5
low = 0
high = 1
and other leave as is

Explanation: the standard deviation of noise which is added is computed by the formula
abs(low-high)*f(eps, delta,...)  , higher eps, delta, lower noise. 
so setting low=high makes noise to be **0**, 

Delta is probability, which is in production should be small (it might be treated as a probability of non-anymizing)
Setting delta = 1, also makes noise to be **0** (maybe not now, because some optimal formulas are not implemented) so it's better to use some more realistic values

The noise should be less than 1

Reply:
Done. Thanks for the explanation!